docs/index.md

```diff
@@ -37,27 +36,31 @@ If you really need to work with older Docker versions, you should look into the
 
 ## TL;DR
 
-```
-# Start your registry
-docker run -d -p 5000:5000 registry:2
+Start your registry
+
+    docker run -d -p 5000:5000 --name registry registry:2
+
+Pull (or build) some image from the hub
+
+    docker pull ubuntu
+
+Tag the image so that it points to your registry
 
-# Pull (or build) some image from the hub
-docker pull ubuntu
+    docker tag ubuntu localhost:5000/myfirstimage
 
-# Tag the image so that it points to your registry
-docker tag ubuntu localhost:5000/myfirstimage
+Push it
 
-# Push it
-docker push localhost:5000/myfirstimage
+    docker push localhost:5000/myfirstimage
 
-# Pull it back
-docker pull localhost:5000/myfirstimage
-```
+Pull it back
 
-## Where to go next
+    docker pull localhost:5000/myfirstimage
 
-Simple as that? Yes. Continue on to read the [overview of the registry](introduction.md).
+Now stop and remove your registry and all data
```

Comment:
That was in reference to the fact the provided instruction will also "remove" the registry container (not just stopping it).
But I agree your version is better either way.